java/com/google/cose/exceptions/CoseException.java

```diff
@@ -0,0 +1,15 @@
+package com.google.cose.exceptions;
+
+/**
+ * This class is used for throwing any exceptions that may occur during parsing and serialization of
+ * byte data into their respective COSE components.
+ */
+public class CoseException extends RuntimeException {
+  public CoseException(final String message) {
+    super(message);
+  }
+
+  public CoseException(final String message, final Exception ex) {
```

Comment:
nit: I think the second parameter should be `Throwable`, not `Exception`.

Reply:
either works fine. But I have updated the code to make use of throwable now. However, I am not making explicit use of either for now.